tests/mockData.js

```diff
@@ -458,50 +476,50 @@ export const permissionSets = [
   },
   {
     name: defaultPermissionSets.ManageReleases.value,
-    action: 'Manage',
-    object: 'Releases',
+    action: uiPermissionsById.manage.title,
+    object: permissionSetObjectTypes.releases,
     description: 'Set of permissions which allows user to manage releases',
     permissions: [
-      { action: 'http', object: { type: 'GET', value: '^/api/management/v1/deployments/artifacts' } },
-      { action: 'http', object: { type: 'GET', value: '^/api/management/v1/deployments/artifacts/[^/]+' } },
-      { action: 'http', object: { type: 'PUT', value: '^/api/management/v1/deployments/artifacts/[^/]+' } },
-      { action: 'http', object: { type: 'DELETE', value: '^/api/management/v1/deployments/artifacts/[^/]+' } },
+      { action: 'http', object: { type: 'GET', value: commonEndpoints.artifacts } },
+      { action: 'http', object: { type: 'GET', value: commonEndpoints.artifactDetails } },
+      { action: 'http', object: { type: 'PUT', value: commonEndpoints.artifactDetails } },
+      { action: 'http', object: { type: 'DELETE', value: commonEndpoints.artifactDetails } },
       { action: 'http', object: { type: 'GET', value: '^/api/management/v1/deployments/artifacts/[^/]+/download' } }
     ]
   },
   {
     name: defaultPermissionSets.ReadUsers.value,
-    action: 'Read',
-    object: 'User management',
+    action: uiPermissionsById.read.title,
+    object: permissionSetObjectTypes.userManagement,
```

Comment:
in a review process I actually went to `src/js/constants/userConstants.js` to check them.